meeting/06-22-2018.md

```diff
@@ -0,0 +1,33 @@
+# Node.js Mentorship Team Meeting 2018-06-22
+
+## Links
+
+* **Recording**: 
+* **GitHub Issue:** https://github.com/nodejs/mentorship/issues/67
+* **Previous Minutes:** 
+
+## Present
+
+- Dan Shaw ([@dshaw](https://github.com/dshaw) - CommComm member, User Feedback champion)
+- Ahmad Bamieh ([@Bamieh](https://github.com/Bamieh))
+- A.J. Roberts ([@detrohutt](https://github.com/detrohutt)
+- Agiri Abraham ([@codeekage](https://github.com/codeekage))
+- Bayo Opesanaya
+- kostas k
+
+### Agenda
+
+Extracted from mentorship-agenda labelled issues and pull requests from the mentorship repo prior to the meeting.
+
+- Follow up discussion for 2018-06-15 meeting #66
+- Identify Mentees Who Could Help Out As "Teaching Assistant"? #49
+
+**TOPIC**
+
```

Comment:
Didn't someone agree to help out with the Mentorship Team? That seems like a good addition to this list. Everything else LGTM.

Reply:
updated

Reply:
🙌

Reply:
I added a commit. I'm happy with everything now. :)